src/test/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/service/GroupsServiceTest.kt

```diff
@@ -1,21 +1,27 @@
 package uk.gov.justice.digital.hmpps.externalusersapi.service
 
+import com.microsoft.applicationinsights.TelemetryClient
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
 import org.mockito.ArgumentMatchers.anyString
 import org.mockito.kotlin.mock
 import org.mockito.kotlin.verify
 import org.mockito.kotlin.whenever
+import uk.gov.justice.digital.hmpps.externalusersapi.jpa.repository.ChildGroupRepository
 import uk.gov.justice.digital.hmpps.externalusersapi.jpa.repository.GroupRepository
 import uk.gov.justice.digital.hmpps.externalusersapi.model.Group
 import uk.gov.justice.digital.hmpps.externalusersapi.security.MaintainUserCheck
 
 class GroupsServiceTest {
   private val groupRepository: GroupRepository = mock()
   private val maintainUserCheck: MaintainUserCheck = mock()
+  private val childGroupRepository: ChildGroupRepository = mock()
+  private val telemetryClient: TelemetryClient = mock()
   private val groupsService = GroupsService(
     groupRepository,
-    maintainUserCheck
+    maintainUserCheck,
+    childGroupRepository,
+    telemetryClient
```

Comment:
Should there be a check for the telemetryclient  being called in your test?

Reply:
Updated